tests/test_regression.py

```diff
@@ -0,0 +1,27 @@
+"""Containing tests that check for regressions in vLLM's behavior.
+
+It should include tests that are reported by users and making sure they
+will never happen again.
+
+"""
+from vllm import LLM, SamplingParams
+
+
+def test_duplicated_ignored_sequence_group():
+    """https://github.com/vllm-project/vllm/issues/1655"""
+
+    sampling_params = SamplingParams(temperature=0.01,
+                                     top_p=0.1,
+                                     max_tokens=256)
+    llm = LLM(model="meta-llama/Llama-2-7b-hf",
```

Comment:
Should probably use a NousResearch models here. But I'm not entirely sure about vllm test infra.

Reply:
Thanks! for this particular test the output weights/types doesn't matter actually since it's just testing the vLLM internal. I changed it to opt-125m. Is there a reason for using NousResearch?

Reply:
oh no reason at all, since I saw you were using llama. Thought there might be specific reason for using llama.